.github/workflows/ci.yml

```diff
@@ -1,34 +1,26 @@
 name: CI
-
 on:
   push:
     branches: [master]
   pull_request:
     branches: [master]
-
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
```

Comment:
Can we preserve this white space? I think the same thing happened with @JordanMartinez using jq

Reply:
I did not find an easy way to insert line breaks with [`yaml`](https://github.com/eemeli/yaml) (in JavaScript) nor [`ruamel.yaml`](https://pypi.org/project/ruamel.yaml/) (in Python). Do you know a formatter we could use? Otherwise we could perhaps hack something with `sed`.

Reply:
I think we hacked something together with `sed` last time. Ultimately the only transformation is to delete the entire `with:` block underneath `setup-purescript`, right?

Reply:
Yes, but I think that will still leave a blank line.

Reply:
I’m going to try the following unless you have another suggestion:

```sh
sed -n '1,/uses: purescript-contrib\/setup-purescript/p' .github/workflows/ci.yml > ci.yml.tmp
echo "" >> ci.yml.tmp
sed -n '/uses: actions\/setup-node/,$p' .github/workflows/ci.yml >> ci.yml.tmp
mv ci.yml.tmp .github/workflows/ci.yml
```

This is less robust than `jq` but at least it preserves the existing formatting.

Reply:
This will also remove the `psa: "0.8.0"` line other libraries have. I assume that `purescript-contrib\/setup-purescript` already installs the last `psa` version anyway?

Reply:
Yes, it already includes the latest `psa`. It's safe to remove.